api/src/test/java/org/openmrs/module/appointments/util/DateUtilTest.java

```diff
@@ -85,6 +85,7 @@ public void shouldReturnEndOfDay() {
 
     @Test
     public void shouldConvertDateToMilliSeconds() throws ParseException {
+    	TimeZone.setDefault(TimeZone.getTimeZone("IST"));
```

Comment:
@mks-d we can have this timezone configuration as part of the mvn build command ?

Reply:
@binduak it is not ok to have Maven builds that are time zone dependent. The whole point of this ticket is to get rid of that situation.

Devs, when writing test cases that are time zone dependent, should be aware of it and should explicitly make this part of their test setups.

Reply:
my bad...i missed going through the card before having a look at PR. Totally makes sense